lib/cronofy/auth.rb

```diff
@@ -22,8 +23,17 @@ def get_token_from_code(code, redirect_uri)
       @client.auth_code.get_token(code, :redirect_uri => redirect_uri).token
     end
 
-    def request
-      @request ||= OAuth2::AccessToken.new(@client, @token)
+    # Public: Refreshes the access token
+    # Returns Hash of token elements to allow client to update in local store for user
+    def refresh!
+      @access_token = access_token.refresh!
```

Comment:
I wonder if this should return the whole `AccessToken`? Similarly `#get_token_from_code` is only returning the `#token` property so there would be no (easy) way for someone to get hold of `#refresh_token` which is pretty important.

Alternatively, it could return `@access_token.to_hash` which returns a similar `Hash`.